README.md

```diff
@@ -120,7 +120,7 @@ Find more examples in the [examples dir](examples).
 The default jnlp agent image used can be customized by adding it to the template
 
 ```groovy
-containerTemplate(name: 'jnlp', image: 'jenkins/jnlp-slave:3.35-5-alpine', args: '${computer.jnlpmac} ${computer.name}'),
+containerTemplate(name: 'jnlp', image: 'jenkins/ jenkins/inbound-agent:3.35-5-alpine', args: '${computer.jnlpmac} ${computer.name}'),
```

Comment:
```suggestion
containerTemplate(name: 'jnlp', image: 'jenkins/inbound-agent:3.35-5-alpine', args: '${computer.jnlpmac} ${computer.name}'),
```
Also, it seems only the latest version has been published (4.3-3), so the tag needs an update.

Also need to coordinate with https://github.com/jenkinsci/kubernetes-plugin/pull/749

Reply:
Yeah it might be better to just do this as part of #749 when 2.222.3 is released.